README.md

```diff
@@ -120,6 +132,9 @@ autoinstall:
     # - curtin in-target -- bash /postinstall.sh
     # - rm /target/postinstall.sh
 
+    # shut-down the host to avoid an infinite installer loop
+    # - shutdown -h now
+
```

Comment:
better to use `shutdown: poweroff` ?

Reply:
My current and still unconfirmed pull request is already a bit out of date. So you're right, using `shutdown: poweroff` is a better choice in this case.